packages/building-rollup/modern-and-legacy-config.js

```diff
@@ -39,74 +46,90 @@ function createConfig(_options, legacy) {
       chunkFileNames: '[name]-[hash].js',
     },
     plugins: [
-      indexHTML({
-        ...(options.indexHTMLPlugin || {}),
```

Comment:
if we leave this option (and the option below) in but don't mention it anymore in the docs, then I think we can release this without a breaking change. what do you think?

Reply:
It would be nice if we can do it with a breaking change, I've updated the pr, can you take another look when you have time?